app/controllers/agenda/minutes.js

```diff
@@ -16,27 +16,29 @@ function renderAttendees(attendees) {
   const { primeMinister, viceMinisters, ministers, secretary } = attendees;
   let secretaryTitle = secretary?.title.toLowerCase() || 'secretaris';
   return `
-    <p><u>AANWEZIG</u></p>
-    <table id="attendees">
-      <tbody>
-        <tr>
-          <td>De minister-president</td>
-          <td>${primeMinister}</td>
-        </tr>
-        <tr>
-          <td>De viceminister-presidenten</td>
-          <td>${viceMinisters.join('<br/>')}</td>
-        </tr>
-        <tr>
-          <td>De Vlaamse ministers</td>
-          <td>${ministers.join('<br/>')}</td>
-        </tr>
-        <tr>
-          <td>De <span id="secretary-title">${secretaryTitle}</span></td>
-          <td><span id="secretary">${mandateeName(secretary)}</span></td>
-        </tr>
-      </tbody>
-    </table>
+    <section data-section="attendees">
+      <p><u>AANWEZIG</u></p>
+      <table>
+        <tbody>
+          <tr>
+            <td>De minister-president</td>
+            <td>${primeMinister}</td>
+          </tr>
+          <tr>
+            <td>De viceminister-presidenten</td>
+            <td>${viceMinisters.join('<br/>')}</td>
+          </tr>
+          <tr>
+            <td>De Vlaamse ministers</td>
+            <td>${ministers.join('<br/>')}</td>
+          </tr>
+          <tr>
+            <td><section data-section="secretary-title"><p>De ${secretaryTitle}</section></p></td>
```

Comment:
using `<td><section ..>De <span> ${secretaryTitle}` created newlines.
This solution looks ok.

Reply:
I already fixed the incorrect closing of html seen in this snippet (open section, open p , close section, close p)